aioify/__init__.py

```diff
@@ -22,10 +22,10 @@ def default_create_name_function(cls):
     return 'create'
 
 
-def aioify(obj, name=None, create_name_function=None):
+def aioify(obj, name=None, create_name_function=None, skip=None):
```

Comment:
Is there a implementation of skip?

Reply:
@yifeikong, yep, `aioify` passes `skip` argument to `module-wrapper`: https://github.com/rominf/module-wrapper/blob/master/module_wrapper/__init__.py#L27